docs/releases/4.2.txt

```diff
@@ -140,6 +140,16 @@ Minor features
 
 * ...
 
+Database backends
+~~~~~~~~~~~~~~~~~
+
+* Third-party database backends can now use callables as values for the
+  ``DatabaseWrapper.data_types`` attribute, the callable should accept a single
+  parameter which is a dictionary of the ``models.Field`` parameters.
```

Comment:
This is not something that I'd document, it's rather a side-effect of the current implementation.